data/DataType/bool_coffee.yaml

```diff
@@ -0,0 +1,7 @@
+keyword: bool
```

Comment:
CoffeeScript doesnt have a bool keyword according to https://ref.pencilcode.net/coffee/keywords.html , which you mentioned earlier

Reply:
Oh, yeah. I forget